drivers/npmx/npmx_shell.c

```diff
@@ -3658,14 +3658,44 @@ static int cmd_timer_config_period_set(const struct shell *shell, size_t argc, c
 	return timer_config_set(shell, argc, argv, TIMER_CONFIG_TYPE_COMPARE);
 }
 
+static const char *shell_err_to_field(npmx_callback_type_t type, uint8_t bit)
+{
```

Comment:
There should be an assertion or an error should probably be returned for wrong values of `type` and `bit`

Reply:
We don't want to use asserts in shell IIRC